### DISCUSSION TITLE
Pro Plan - More Granular User Roles

### DISCUSSION THREAD

**Discussion by benmkramer** [vercel/community#1011] [Idea]
Being on the pro plan, any user I invite to my application will have access to all configuration, including environment variables for the production app.

As we have some part time contractors who really have no need to accessing our production DB or having keys to other services, but could benefit from having preview deployments and seeing builds for debugging, it would be great to use some of the more constrained roles that the enterprise plan looks to offer without having to go to the enterprise plan (I suspect its a big jump in pricing, for many features I don't need yet). 

Is this something in the works? It would be great to have in the pro plan, even if it costed a bit more / user / month or was some other kind of upcharge.

**Comment by amyegan** [comment]
Thanks for bringing up this concern @benmkramer! This does seem like something we would want to offer in the future. I'm sure there are layers of complexity to it that I'm not aware of, so I'm sharing it with the team to investigate and prioritize.
